tests/linux-sandbox.sh

```diff
@@ -29,24 +31,29 @@ nix store ls -R -l $outPath | grep foobar
 nix store cat $outPath/foobar | grep FOOBAR
 
 # Test --check without hash rewriting.
-nix-build dependencies.nix --no-out-link --check --sandbox-paths /nix/store
+nix-sandbox-build dependencies.nix --check
 
 # Test that sandboxed builds with --check and -K can move .check directory to store
-nix-build check.nix -A nondeterministic --sandbox-paths /nix/store --no-out-link
+nix-sandbox-build check.nix -A nondeterministic
 
-(! nix-build check.nix -A nondeterministic --sandbox-paths /nix/store --no-out-link --check -K 2> $TEST_ROOT/log)
-if grepQuiet 'error: renaming' $TEST_ROOT/log; then false; fi
+expectStderr 104 nix-sandbox-build check.nix -A nondeterministic --check -K > $TEST_ROOT/log
```

Comment:
Where does `104` come from? I am curious, but it might also be good to clarify in the tests and/or code.

Reply:
But of course, everything is documented _somewhere_ in nix ;-).

In the source code:
https://github.com/NixOS/nix/blob/3c618c43c6044eda184df235c193877529e951cb/src/libstore/build/worker.cc#L471-L496

And in the manual:
https://nixos.org/manual/nix/stable/command-ref/nix-store/realise.html

Of course, these things can only be found if you know they exist.

Reply:
Great sleuthing!